automatic/es/README.md

```diff
@@ -0,0 +1,7 @@
+# <img src="https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-packages@cf277846cf6b5436461ebd5f21b00efd67ee9910/icons/everything.png" width="48" height="48"/> [everything](https://chocolatey.org/packages/everything)
+
+ES is a command line interface to search Everything from a command prompt.
+
+"Everything" is an administrative tool that locates files and folders by filename instantly for Windows.
+Unlike Windows search "Everything" initially displays every file and folder on your computer (hence the name "Everything").
+You type in a search filter to limit what files and folders are displayed.
```

Comment:
Please include the section `Features` for the application.

See https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md#123-informative-description

Reply:
Done